packages/richtext-lexical/src/features/indent/client/plugins/index.tsx

```diff
@@ -3,72 +3,59 @@
 import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext.js'
 import { TabIndentationPlugin } from '@lexical/react/LexicalTabIndentationPlugin'
 import { mergeRegister } from '@lexical/utils'
-import { COMMAND_PRIORITY_LOW, FOCUS_COMMAND, KEY_ESCAPE_COMMAND } from 'lexical'
+import { COMMAND_PRIORITY_NORMAL, FOCUS_COMMAND, KEY_ESCAPE_COMMAND } from 'lexical'
 import { useEffect, useState } from 'react'
 
 import type { PluginComponent } from '../../../typesClient.js'
 
 export const IndentPlugin: PluginComponent<undefined> = () => {
   const [editor] = useLexicalComposerContext()
 
-  const [tabIndentEnabled, setTabIndentEnabled] = useState<boolean>(true)
+  const [firefoxFlag, setFirefoxFlag] = useState<boolean>(false)
```

Comment:
I'm not sure I agree with the name change here as it could create confusion. This code will run in all browsers, we are not conditionally running it just for firefox.

Reply:
Yes, you are right. The problem is that `tabIndentEnabled` was not descriptive anymore because tabIndent is always enabled. What this flag does is indicate when the indent behavior should be normalized. In non-firefox browsers it is irrelevant since it works fine by default, and I preferred to make it clear to someone reading the plugin for the first time that this is a Firefox-only property.
I can't think of a better name. Perhaps an explanatory comment would be nice.